src/main/java/de/tum/in/www1/artemis/service/QuizExerciseImportService.java

```diff
@@ -104,8 +104,13 @@ private void copyQuizQuestions(QuizExercise importedExercise, QuizExercise newEx
             }
             else if (quizQuestion instanceof DragAndDropQuestion dndQuestion) {
                 if (dndQuestion.getBackgroundFilePath() != null) {
+                    URI backgroundFilePublicPath = URI.create(dndQuestion.getBackgroundFilePath());
+                    URI backgroundFileIntendedPath = URI.create("/api/" + FilePathService.getDragAndDropBackgroundFilePath() + "/");
```

Comment:
Still incorrect. This is the URI prefix combined with the file system path. Same below

Reply:
Addressed by [be06cebea3bdf2a0e7edd0ff8c3c98521a5bee33](https://github.com/ls1intum/Artemis/pull/7170/commits/be06cebea3bdf2a0e7edd0ff8c3c98521a5bee33).